core/src/main/java/io/questdb/cairo/map/FastMap.java

```diff
@@ -340,6 +364,64 @@ private FastMapValue asNew(BaseKey keyWriter, int index, int hashCode, FastMapVa
         return valueOf(keyWriter.startAddress, keyWriter.appendAddress, true, value);
     }
 
+    private void mergeFixedSizeKey(FastMap srcMap, GroupByFunctionsUpdater mergeFunc) {
```

Comment:
nit: cleaner to have Map interface, not adding dependency here to `GroupByFunctionsUpdater`

Reply:
Good idea. Done in 8c8f37e941ea4ef208a0f0261cd3c5d7079b57f5